### PR TITLE
fix some spelling mistakes

### PR DESCRIPTION
**Pull request by hwdef**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by geeknoid** [comment]
Hi,

The files in common-protos/* are all imported from elsewhere, so they shouldn't be changed as we want to keep them completely in sync with the originals.

Thanks.

File: common-protos/google/api/expr/v1alpha1/eval.proto [hunk 1]
```diff
@@ -28,7 +28,7 @@ option java_package = "com.google.api.expr.v1alpha1";
 
 // The state of an evaluation.
 //
-// Can represent an inital, partial, or completed state of evaluation.
+// Can represent an initial, partial, or completed state of evaluation.
```
Comment (howardjohn) [inline common-protos/google/api/expr/v1alpha1/eval.proto]: these files are copied from istio/common-files repo which in turn copies them from some google repo, so I don't think we should change these. Same applies for others under common-protos.

the other files are fine though

Comment (hwdef) [reply common-protos/google/api/expr/v1alpha1/eval.proto]: I'm sorry about that, and i fixed it.

**Comment by hwdef** [comment]
@geeknoid 
Thanks for your suggestion.

**Comment by hwdef** [comment]
@geeknoid 
Can you help me check what is wrong with this RP?
Two days, it has not merged.

**Pull request merged**